packages/cli/src/commands/init.ts

```diff
@@ -36,11 +38,9 @@ export const command = (
       const workspace = await initLocalWorkspace(baseDir, workspaceName, defaultEnvName)
       const workspaceTags = await getWorkspaceTelemetryTags(workspace)
       cliTelemetry.success(workspaceTags)
-      stdout.write(
-        Prompts.initCompleted(workspace.name, baseDir)
-      )
+      outputLine(Prompts.initCompleted(workspace.name, baseDir), { stdout })
     } catch (e) {
-      stderr.write(Prompts.initFailed(e.message))
+      errorOutputLine(Prompts.initFailed(e.message), { stderr })
```

Comment:
nit: instead of deconstructing the `CliOutput` in the parameter list:
```typescript
{ stdout, stderr }: CliOutput
```
why not keep it as
```typescript
output: CliOutput
```
and then both calls here can be
```suggestion
      errorOutputLine(Prompts.initFailed(e.message), output)
```
just seems a little cleaner than deconstructing the object just to re-build it for the function call.

this repeats a few times in the code